services/121-service/src/payments/fsp-integration/intersolve-visa/intersolve-visa.api.service.ts

```diff
@@ -23,8 +23,9 @@ import { IssueTokenResultDto } from '@121-service/src/payments/fsp-integration/i
 import { IssueTokenDto } from '@121-service/src/payments/fsp-integration/intersolve-visa/dto/internal/issue-token.dto';
 import { TransferDto } from '@121-service/src/payments/fsp-integration/intersolve-visa/dto/internal/transfer.dto';
 import {
-  IntersolveBlockWalletDto,
-  IntersolveBlockWalletResponseDto,
+  BlockReasonEnum,
+  IntersolveApiBlockWalletResponseDto,
+  UnblockReasonEnum,
 } from '@121-service/src/payments/fsp-integration/intersolve-visa/dto/intersolve-block.dto';
```

Comment:
To do: refactor this DTO file (file location, file name, split into parts? Combining DTOs?)